### PR TITLE
fix: Swen testnet deploy

### PR DESCRIPTION
**Pull request by SwenSchaeferjohann**
- hosted relayer runs on test net

### CODE REVIEW

**Comment by SwenSchaeferjohann** [comment]
tangentially related: 

- test: payments app use testnet + indexer
- cli with testnet (blocked)

File: relayer/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "dependencies": {
     "@coral-xyz/anchor": "^0.28.0",
     "@coral-xyz/borsh": "^0.28.0",
-    "@lightprotocol/zk.js": "file:../light-zk.js",
+    "@lightprotocol/zk.js": "0.3.2-alpha.13",
```
Comment (ananas-block) [inline relayer/package.json:20]: We need this to import by path for testing. What about doing relayer release branches which have the actual version number and will be deployed?

File: relayer/src/config.ts [hunk 2]
```diff
@@ -36,11 +37,20 @@ export enum TransactionType {
...
-export const RPC_URL = process.env.RPC_URL;
+export const RPC_URL = process.env.RPC_URL!;
 console.log("RPC_URL", RPC_URL);
 
+export const PORT = process.env.DB_PORT!;
+export const PASSWORD = process.env.PASSWORD!;
+export const HOST = process.env.HOSTNAME!;
+
+console.log("PORT", PORT);
+console.log("PASS: ", PASSWORD);
```
Comment (ananas-block) [inline relayer/src/config.ts:51]: let's remove at least this log

File: relayer/tests/browser_test.ts [hunk 3]
```diff
@@ -61,20 +62,20 @@ describe("Browser tests", () => {
...
+  const connection = new Connection(RPC_URL, "confirmed");
 
   before(async () => {
     poseidon = await circomlibjs.buildPoseidonOpt();
 
     await createTestAccounts(connection);
 
     const relayerRecipientSol = SolanaKeypair.generate().publicKey;
-    await connection.requestAirdrop(relayerRecipientSol, 2_000_000_000);
+    await connection.requestAirdrop(relayerRecipientSol, 2 * AIRDROP_DECIMALS);
```
Comment (ananas-block) [inline relayer/tests/browser_test.ts:74]: happy for a SOL_DECIMALS constant in zk.js but I don't think it's good to start putting such constants in many different places.

**Review by ananas-block — CHANGES_REQUESTED** [review]

File: relayer/tests/browser_test.ts [hunk 4]
```diff
@@ -61,32 +62,32 @@ describe("Browser tests", () => {
...
+      lamports: 9e8,
       recipientPublicKey: relayer.publicKey,
     });
 
     // TODO: This will only work as long as .env keys don't change
     RELAYER = new Relayer(
       new PublicKey("EkXDLi1APzu6oxJbg5Hnjb24kfKauJp1xCb5FAUMxf9D"),
       new PublicKey("AV3LnV78ezsEBZebNeMPtEcH1hmvSfUBC5Xbyrzqbt44"),
-      new BN(100000),
+      new BN(5000),
```
Comment (ananas-block) [inline relayer/tests/browser_test.ts:86]: why did you reduce the relayer fee?
(We do 100k in all our tests)

**Review by ananas-block — APPROVED** [review]

**Pull request merged**